service/conf/routes

```diff
@@ -131,6 +131,8 @@ POST   /v1/audit/history                       @controllers.audit.AuditLogContro
 POST  /v1/user/skill/add 						@controllers.skills.SkillController.addSkill()
 POST  /v1/user/skill/read						@controllers.skills.SkillController.getSkill()
```

Comment:
Add temporary paths for getSkill and listSkills in UserSkillController. After testing we will change the previous paths to latest paths.